src/base/utils/fs.cpp

```diff
@@ -518,3 +527,54 @@ QString Utils::Fs::cacheLocation()
         locationDir.mkpath(locationDir.absolutePath());
     return location;
 }
+
+Utils::Fs::FileProcessingGuard::FileProcessingGuard(const QString &path)
+    : m_path{path}
```

Comment:
Uhm... I had to discuss this before, but what's the difference:
```C++
m_member1{value1}
m_member2(value2)
```
Why do you want space int the first case, but not in the second one?